account_invoice_user_authorization_payment/views/account_move_views.xml

```diff
@@ -3,25 +3,30 @@
 <odoo>
 
     <record id="user_authorization_payment_tree" model="ir.ui.view">
-        <field name="name">account.move.tree (User Authorization Payment)</field>
+        <field name="name">account.move.tree (User Authorization Customer Payment)</field>
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_in_invoice_tree"/>
-        <field name="groups_id" eval="[(4, ref('account.group_account_user'))]"/>
+        <field name="groups_id" eval="[
+            (4, ref('account_invoice_user_authorization_payment.group_user_payment'))]"/>
         <field name="arch" type="xml">
             <xpath expr="//field[@name='name']" position="after">
-                <field name="create_uid" />
+                <field name="create_uid" widget="many2one_avatar_user" optional="show" />
+            </xpath>
+            <xpath expr="//field[@name='create_uid']" position="after">
```

Comment:
No need for this extra `xpath` section: simply add the new `<field>` lines together.